db/migrate/20221017163348_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :post, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Same here, Please edit this reference because it's supposed to be `author_id` not `user_id` as a foreign key.